### PR TITLE
Add solution item accordion

### PR DESCRIPTION
**Pull request by jmhauck**
This will be used to show what's included in the deploy app

### CODE REVIEW

File: src/components/solution-item-accordion/solution-item-accordion.tsx [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    let updatedType = templateInfo.type;
+    if (templateInfo.type === "Feature Service") {
+      updatedType = templateInfo.typeKeywords.indexOf("View Service") > -1 ?
+        "Feature Layer (hosted, view)" : "Feature Layer (hosted)";
+    }
+
+    if (templateInfo.type === "Web Mapping Application") {
+      updatedType = templateInfo.typeKeywords.indexOf("configurableApp") > -1 ?
+        "Instant App" : updatedType;
+    }
```
Comment (MikeTschudi) [inline src/components/solution-item-accordion/solution-item-accordion.tsx:296]: Do none of our solutions use Web Mapping Applications anymore--they're all Instant Apps?

File: src/components/solution-item-accordion/test/solution-item-accordion.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { newE2EPage } from '@stencil/core/testing';
+
+xdescribe('solution-item-accordion', () => {
```
Comment (MikeTschudi) [inline src/components/solution-item-accordion/test/solution-item-accordion.e2e.ts:19]: Can we at least test that the component renders?

File: src/components/solution-item-accordion/test/solution-item-accordion.spec.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { newSpecPage } from '@stencil/core/testing';
+import { SolutionItemAccordion } from '../solution-item-accordion';
+
+xdescribe('solution-item-accordion', () => {
```
Comment (MikeTschudi) [inline src/components/solution-item-accordion/test/solution-item-accordion.spec.tsx:20]: Again, a non-working test?  Maybe a test to verify that the display order is honored?

**Review by MikeTschudi — APPROVED** [review]

File: src/components/solution-item-accordion/solution-item-accordion.tsx (hunk 1 above)
Comment (jmhauck) [reply src/components/solution-item-accordion/solution-item-accordion.tsx:296]: Only instant app if it has the "configurableApp" typeKeyword.

File: src/components/solution-item-accordion/test/solution-item-accordion.e2e.ts (hunk 2 above)
Comment (jmhauck) [reply src/components/solution-item-accordion/test/solution-item-accordion.e2e.ts:19]: I can add a simple test...was more focused on making sure that I could get the full pattern down to show this in both the deploy app and item details page.

**Comment by jmhauck** [comment]
Removing tests for now...will add back after we finalize the component

**Pull request merged**